tests/verifyMongodbSharding_test.js

```diff
@@ -0,0 +1,44 @@
+Feature('MongoDB Sharding tests');
+
+const { adminPage } = inject();
+const pmmFrameworkLoader = `bash ${adminPage.pathToFramework}`;
+const exactMongoDbInfo = new DataTable(['version', 'edition']);
+
+// exactMongoDbInfo.add(['4.0', 'Community']);
+exactMongoDbInfo.add(['4.2', 'Community']);
```

Comment:
this would make our pipeline too complicated, I think just test it with latest version of MongoDB, if it doesn't work this is not a customer blocker

Reply:
Fixed